pkg/reconciler/mtbroker/trigger/controller.go

```diff
@@ -66,6 +71,28 @@ func NewController(
 
 	triggerInformer.Informer().AddEventHandler(controller.HandleAll(impl.Enqueue))
```

Comment:
we should also filter the trigger changes based on which broker it points to.

Reply:
@n3wscott can you explain more what you mean? doesn't the label selector filter by the broker it points to? Or am I misunderstanding.

Reply:
This broker impl only needs triggers for brokers. We filter broker changes for triggers but we should do the other side too?